internal/backends/mcu/lib.rs

```diff
@@ -127,8 +130,15 @@ mod the_backend {
         }
         fn free_graphics_resources<'a>(
             &self,
-            _items: &mut dyn Iterator<Item = Pin<i_slint_core::items::ItemRef<'a>>>,
+            items: &mut dyn Iterator<Item = Pin<i_slint_core::items::ItemRef<'a>>>,
         ) {
+            super::PARTIAL_RENDERING_CACHE.with(|cache| {
+                for item in items {
+                    let cache_entry =
+                        item.cached_rendering_data_offset().release(&mut cache.borrow_mut());
+                    drop(cache_entry);
+                }
+            });
```

Comment:
Technically this I think should also mark those regions as dirty, but since they're relative I think it would require extra calculations to get the absolute value.

For the popups this isn't needed due to the `close_popup` logic, but we might need to extend this for other dynamic areas?